api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -81,7 +81,7 @@ public DbInstitution setOrganizationTypeOtherText(String organizationTypeOtherTe
     return this;
   }
 
-  @OneToMany(mappedBy = "institution", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+  @OneToMany(mappedBy = "institution", cascade = CascadeType.ALL)
```

Comment:
EAGER fetching became problematic when this was fetched transitively by VerifiedInstitutionalAffiliation.

TODO: try to understand the Hibernate fetching model better to understand why this was the case